### DISCUSSION TITLE
0.3.0 release

### DISCUSSION THREAD

**Discussion by 0323pin** [leftwm/leftwm#755] [General]
Hi,

Is it possible to get a ChangeLog for the 0.3.0 release from a few hours ago?

Thanks!

**Comment by VuiMuich** [comment]
Hi,

I don't really see, why you would want this for this specific time frame, when the latest commit was almost a month back.
We used to write the actual changelogs in the process of creating a release tag typically.

**Comment by 0323pin** [comment]
I can easily create one with the most relevant commits going from 0.2.11 to 0.3.0, after all I've been running the WM on git-HEAD all this time.
It's just one thing we do on NetBSD when updating packages but, don't worry, I'll fix that and merge the update rather soon.

**Comment by 0323pin** [comment]
@VuiMuich merged, https://mail-index.netbsd.org/pkgsrc-changes/2022/04/25/msg253122.html

**Reply by VuiMuich** [reply]
🤦🏼‍♂️ I totally missed we actually released 3.0.0, sorry.

**Reply by 0323pin** [reply]
😄  No worries, I was surprised to see a release as well.
BTW, I'm again running on the `lefthk` branch. As long as I don't reload, it works fine.
